src/components/IntroEditor/index.js

```diff
@@ -0,0 +1,150 @@
+import React from "react";
+import CustomPropTypes from "custom-prop-types";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import { flip, partial, flowRight } from "lodash";
+import { connect } from "react-redux";
+
+import { raiseToast } from "redux/toast/actions";
+
+import MainCanvas from "components/MainCanvas";
+import IconButtonDelete from "components/IconButtonDelete";
+import withEntityEditor from "components/withEntityEditor";
+import Button from "components/Button";
+import IconText from "components/IconText";
+import AddPage from "components/QuestionnaireDesignPage/icon-add-page.svg?inline";
+import RichTextEditor from "components/RichTextEditor";
+import { Toolbar, Buttons } from "components/EditorToolbar";
+import FadeTransition from "components/FadeTransition";
+import { TransitionGroup } from "react-transition-group";
+
+import { colors, radius } from "constants/theme";
+
+import sectionFragment from "graphql/fragments/section.graphql";
+import withDeleteSectionIntro from "containers/enhancers/withDeleteSectionIntro";
+
+const StyledMainCanvas = styled(MainCanvas)`
+  padding: 0 2em 1em;
+`;
+
+const AddIntroButton = styled(Button)`
+  width: 100%;
+  padding: 0.8em 1.8em 0.8em 0.8em;
+  border: 1px solid ${colors.bordersLight};
+`;
+
+const IntroToolbar = styled(Toolbar)`
+  border: 1px solid ${colors.bordersLight};
+  border-bottom: 0;
+  background-color: ${colors.white};
+  border-radius: ${radius} ${radius} 0 0;
+`;
+
+export const IntroCanvas = styled.div`
+  padding: 0.8em 1.8em;
```

Comment:
Bit too much padding here:

```suggestion
  padding: 0 1em;
```